### DISCUSSION TITLE
Checked in Reminder

### DISCUSSION THREAD

**Discussion by ColeStansbury** [sandboxnu/office-hours#546] [Ideas]
Hey Cole here! TA from CS3500 and MSCS student.

Soooooo... I tend to be really bad about forgetting I'm checked into the OH app. Not sure how hard this might be but any chance you could design a feature that allows me to opt in for auto signing out or email reminders or something? I keep confusing students by doing this :(

Thank you!

**Comment by liustanley** [comment]
Hi Cole! Thanks for the feedback, we've actually recently implemented a feature that checks out TA's every night in case they forgot to check out at the end of the queue. Hope this helps to resolve your problem!
